src/CHANGES

```diff
@@ -1,5 +1,11 @@
+Changes 2.17.11, 2022-12-19
+---------------------------
+From: fweimer-rh
 * Future C compilers are likely to reject implicit function declarations by default. This language feature was officially removed in 1999. @ffwmeier-rh
 
+From: lukerg
+* RouterUpTime accepts & uses colon options
+
 Changes 2.17.11, 2022-??-??
 ---------------------------
```

Comment:
```suggestion
From: fweimer-rh
* Future C compilers are likely to reject implicit function declarations by default. This language feature was officially removed in 1999. @ffwmeier-rh

From: lukerg
* RouterUpTime accepts & uses colon options

```

As these changes have not yet been released, there is no version number to go with them :)